open_samus_returns_rando/files/templates/metroid_template.lua

```diff
@@ -0,0 +1,43 @@
+Metroid = {}
+function Metroid.Dummy()
+end
+
+function Metroid.RemoveMetroid(_ARG_0_)
+    local spawnGroupName = CurrentScenario.currentMetroidSpawngroup
+    if spawnGroupName ~= nil then
+        -- GUI.LaunchMessage(spawnGroupName, "Metroid.Dummy", "")
+
+        local spawnGroup =  Game.GetEntity(spawnGroupName)
+        if spawnGroup ~= nil and spawnGroup.SPAWNGROUP ~= nil then
+            spawnGroup.SPAWNGROUP:DisableSpawnGroup()
+        else 
+            GUI.LaunchMessage("Oops 1" .. spawnGroupName, "Metroid.Dummy", "")
+            return
+        end
+
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_dead", "b", true)
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_deaths", "i", 1)
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_enabled", "b", false)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_dead", "b", true)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_deaths", "i", 1)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_enabled", "b", false)
```

Comment:
Was this supposed to be changed?

Reply:
no :(

Reply:
I don't get its purpose then

Reply:
Well, I just tested stuff and things and forgot to remove it

Reply:
Ah okay so not intentional

Reply:
Removed it